frontend/src/AccountCreation/SignUpProfessor.tsx

```diff
@@ -24,54 +26,101 @@ const SignUpProfessor = () => {
+                  color: "black",
+                  font: "Hammersmith One",
+                  fontSize: 30,
+                  fontWeight: "bolder",
+                  marginRight: 20,
+                }}
+              >
+                {courseCode}
+              </Form.Label>
+            </Form.Group>
+            <br />
             <br />
-            <Form.Label
-              style={{
-                textAlign: "center",
-                justifyContent: "center",
-                color: "black",
-                font: "Hammersmith One",
-                fontSize: 30,
-                fontWeight: "bolder",
+            <Button
+              onClick={() => {
+                navigator.clipboard.writeText(courseCode);
               }}
+              variant="outline-secondary"
+              type="button"
+              style={{ borderRadius: 100 }}
             >
-              XXX-XXX-XXX
-            </Form.Label>
-          </Form.Group>
+              <label>Copy</label>
+            </Button>
+          </Stack>
           <Form.Group className="mb-3" controlId="formCourseCode">
-            <Form.Label>First and Last Name</Form.Label>
+            <Form.Label className="signUpFormText">
+              First and Last Name
+            </Form.Label>
             <Form.Control
+              className="signUpForm"
               type="text"
               name="course code"
               placeholder="Joe Bruin"
             />
           </Form.Group>
           <Form.Group className="mb-3" controlId="formCourseCode">
-            <Form.Label>UID</Form.Label>
+            <Form.Label className="signUpFormText">UID</Form.Label>
             <Form.Control
+              className="signUpForm"
               type="text"
               name="course code"
-              placeholder="XXXXXXXXX"
+              placeholder="---------"
             />
           </Form.Group>
           <Form.Group className="mb-3" controlId="formCourseName">
-            <Form.Label>Course Name</Form.Label>
-            <Form.Control type="text" name="course name" />
+            <Form.Label className="signUpFormText">Course Name</Form.Label>
+            <Form.Control
+              className="signUpForm"
+              type="text"
+              name="course name"
+            />
           </Form.Group>
           <Form.Group className="mb-3" controlId="formPassword">
-            <Form.Label>Password</Form.Label>
-            <Form.Control type="password" name="password" />
+            <Form.Label className="signUpFormText">Password</Form.Label>
+            <Form.Control
+              className="signUpForm"
+              type="password"
+              name="password"
+            />
           </Form.Group>
           <Form.Group className="mb-3" controlId="formPassword">
-            <Form.Label>Re-enter Password</Form.Label>
-            <Form.Control type="password" name="password" />
+            <Form.Label className="signUpFormText">
```

Comment:
https://getbootstrap.com/docs/5.0/forms/validation/ <- could be cool to add validation styles to show if passwords aren't matching!! (low priority tho just wanted to flag this if you have time but we can merge without adding this)

Reply:
I have form validation in my other PR